AUTHORS.md

```diff
@@ -0,0 +1,259 @@
+* [Alex Conlin](https://github.com/alexconlin)
+* [Alex Gaynor](https://github.com/alex)
+* [Alex Halderman](https://github.com/jhalderm)
+* [Alex Jordan](https://github.com/strugee)
+* [Amjad Mashaal](https://github.com/TheNavigat)
+* [Andrew Murray](https://github.com/radarhere)
+* [Anselm Levskaya](https://github.com/levskaya)
+* [Antoine Jacoutot](https://github.com/ajacoutot)
+* [asaph](https://github.com/asaph)
+* [Axel Beckert](https://github.com/xtaran)
+* [Bas](https://github.com/Mechazawa)
+* [benbankes](https://github.com/benbankes)
+* [Ben Irving](https://github.com/benileo)
+* [Benjamin Kerensa](https://github.com/bkerensa)
+* [Benjamin Neff](https://github.com/SuperTux88)
+* [Benjamin Piouffle](https://github.com/Betree)
+* [Ben Ubois](https://github.com/benubois)
+* [Ben Wolfe](https://github.com/bwolfe)
+* [Bigfish](https://github.com/bwolfe)
+* [Blake Griffith](https://github.com/cowlicks)
+* [Brad Warren](https://github.com/bmw)
+* [Brandon Kraft](https://github.com/kraftbj)
+* [Brandon Kreisel](https://github.com/kraftbj)
+* [Ceesjan Luiten](https://github.com/quinox)
+* [Chad Whitacre](https://github.com/whit537)
+* [Chhatoi Pritam Baral](https://github.com/pritambaral)
+* [Chris Johns](https://github.com/ter0)
+* [Chris Lamb](https://github.com/lamby)
+* [chrismarget](https://github.com/chrismarget)
+* [Christian Gärtner](https://github.com/ChristianGaertner)
+* [Christian Rosentreter](https://github.com/the-real-tokai)
+* [Christopher Brown](https://github.com/chbrown)
+* [Christopher Manning](https://github.com/christophermanning)
+* [Christoph Kisfeld](https://github.com/chk1)
+* [Clif Houck](https://github.com/ClifHouck)
+* [Cooper Quintin](https://github.com/cooperq)
+* [Corey Farwell](https://github.com/frewsxcv)
+* [Craig Smith](https://github.com/dashaxiong)
+* [Damian Poddebniak](https://github.com/duesee)
+* [Damien Nozay](https://github.com/dnozay)
+* [Damien Tournoud](https://github.com/damz)
+* [DanCld](https://github.com/DanCld)
+* [Daniel Albers](https://github.com/AID)
+* [Daniel Aleksandersen](https://github.com/da2x)
+* [Daniel Convissor](https://github.com/convissor)
+* [Daniel Huang](https://github.com/dhuang)
+* [David cz](https://github.com/dave-cz)
+* [Dave Guarino](https://github.com/daguar)
+* [David Dworken](https://github.com/ddworken)
+* [David Kreitschmann](https://kreitschmann.de)
+* [David Xia](https://github.com/davidxia)
+* [Devin Howard](https://github.com/devvmh)
+* [dokazaki](https://github.com/dokazaki)
+* [Dominic Cleal](https://github.com/domcleal)
+* [Dominic Lüchinger](https://github.com/dol)
+* [Douglas José](https://github.com/douglasjose)
+* [Eric Rescorla](https://github.com/ekr)
+* [Eric Engestrom](https://github.com/1ace)
+* [Eric Wustrow](https://github.com/ewust)
+* [Erica Portnoy](https://github.com/ohemorange)
+* [Erik Rose](https://github.com/erikrose)
+* [Eugene Kazakov](https://github.com/xgin)
+* [Giovanni Pellerano](https://github.com/evilaliv3)
```

Comment:
Not to nit this point too much, but the sorted order still seems off. This line is one example.